docs/plone-deployment/new-project.md

```diff
@@ -142,3 +141,18 @@ When the process completes successfully, it will exit with a message similar to
 ```console
 ✨  Done in 98.97s.
 ```
+
+Remember, having the repository on GitHub is currently a requirement to build the Docker images automatically; this might be a good moment to do so in case you had not yet.
```

Comment:
Provide a reference, and tweak wording.

This paragraph should be moved up almost to the top, after the page title at line 10, to become its introduction. I think is is good to first explain what we are going to do in this chapter, and why these choices were made.

```suggestion
As mentioned in the {doc}`intro`'s Training Choices, for this Training, GitHub is a requirement to build the Docker images automatically.
The steps in this training may be adapted to other providers, including GitLab.
```